### ISSUE TITLE
Beatmap lost ranked status

### ISSUE THREAD

**Issue by Phippe** [ppy/osu-stable-issues#593]
I played [this beatmap](https://osu.ppy.sh/beatmapsets/356821#osu/785097) and when I finished I got an error saying the online ranking for this beatmap wasn't available, maybe it had lost its ranked status.

[Video](https://streamable.com/whv0yv)

I can't remember ever editing this beatmap to make it lose its ranked status, but I could be wrong. If there are any logs I can provide please tell me, and if this issue is out of scope then so be it, wasn't an fc anyway :P.

**Comment by smoogipoo** [comment]
Did you ever edit this map in the editor?

**Comment by Phippe** [comment]
I don't think so, I don't see why would do that on this map. Then again I haven't returned to this map in a while, so I might have edited last time. Unfortunately, I cannot guarantee that I didn't open it in the editor :/

**Comment by stevefan1999-personal** [comment]
https://user-images.githubusercontent.com/29133953/113404977-a3ca5800-93db-11eb-952e-95e3c591776e.mp4
Confirmable
